test/JDBC/upgrade/16_9/schedule

```diff
@@ -607,7 +607,8 @@ BABEL-NUMERIC_EXPR
 test_conv_float_to_varchar_char
 BABEL-5467
 BABEL_OBJECT_RESOLUTION_IN_ROUTINES
-test_constraint_like
+test_constraint_like-17-5-or-16-9
 round_return_type_test
 alter-view
 test_conv_int_to_varbinary_binary
+test_like_index
```

Comment:
isnt this same?

Reply:
As discussed offline, this is because this input file has output for these specific versions